app/controllers/application_controller.rb

```diff
@@ -11,7 +11,7 @@ class ApplicationController < ActionController::Base
 
   ERRORS = {
     not_logged_in: "Please log in to see this page.", # used in ApplicationController's require_seller_login
-    already_in_cart: "This item is already in your cart!", # used in OrderController's add_to_cart
+    already_in_cart: "This item is already in your cart!",
```

Comment:
It's not being used anywhere, and it is a DB error now, so I don't have any errors to add to this.